test/component/spec.handler.spec.js

```diff
@@ -82,4 +82,18 @@ describe('Spec Handler', () => {
       });
   });
 
+  it('sleep in spec handler', async () => {
+    await pactum
+      .spec('get users')
+      .expectStatus(200)
+      .expectJson([
+        {
+          id: 1
+        },
+        {
+          id: 2
+        }
+      ])
+      .sleep(2000);
```

Comment:
reduce sleep duration to allow tests to be faster
```suggestion
      .sleep(10);
```